src/Groups/homomorphisms.jl

```diff
@@ -285,6 +288,50 @@ function isisomorphic(G::GAPGroup, H::GAPGroup)
   end
 end
 
+"""
+    isomorphic_perm_group(G::GAPGroup)
+Return a group of permutation `H` and an isomorphism `f` from `G` to `H`.
+"""
+function isomorphic_perm_group(G::GAPGroup)
+   f = GAP.Globals.IsomorphismPermGroup(G.X)
+   if f==GAP.Globals.fail
+      throw(ArgumentError("Could not convert group into a permutation group"))
+   else
```

Comment:
I suggest to check the error conditions at the start, and then proceed normally, on the "top level" of the function, not within an "else" block.

And in fact, I'd suggest to follow this pattern for argument checks:
```suggestion
   f!=GAP.Globals.fail || throw(ArgumentError("Could not convert group into a permutation group"))
```